internal/rpmver/rpmver.go

```diff
@@ -128,6 +128,7 @@ func Parse(v string) (Version, error) {
 // and what's just another version segment.
 var architectures = map[string]struct{}{
 	".aarch64": {},
+	".i386":    {},
```

Comment:
This seems off-topic, and possibly unwanted? I don't think there are any non-EOL distributions that build for `i386`.

Reply:
You'd be surprised what one could find on a real workstation system: https://www.brother.co.uk/en/support/mfc9140cdn/downloads.

Reply:
The `i386` arch is now gone.